src/utils/index.js

```diff
@@ -5,7 +5,19 @@ const dateFormat = dateString => {
 }
 
 const numberFormat = x => {
-  return Intl.NumberFormat('FR-fr').format(x)
+  return Intl.NumberFormat('FR-fr', {
+    maximumSignificantDigits: 21
+  }).format(x)
+}
+
+const textToNumberFormat = (val, locale) => {
+  const group = new Intl.NumberFormat(locale).format(1111).replace(/1/g, '')
```

Comment:
j'ai besoin d'une explication sur ce qui se passe ici stp.

Reply:
j'ai trouvé la fonction ici: https://stackoverflow.com/questions/29255843/is-there-a-way-to-reverse-the-formatting-by-intl-numberformat-in-javascript
ce que je comprend c'est que ça formate 1111 en number local 'FR' donc => 1 111
sur quoi ça remplace les '1' par '' => '' ''''''
on en fait une regex qu'on remplace par '' ce qui a pour effet de retirer le séparateur de millier
principe similaire pour remplacer les , par le .

Reply:
j'ai modifié par beaucoup plus simple...